app/models/case_log.rb

```diff
@@ -48,6 +48,8 @@ class CaseLog < ApplicationRecord
   scope :for_organisation, ->(org) { where(owning_organisation: org).or(where(managing_organisation: org)) }
 
   enum status: { "not_started" => 0, "in_progress" => 1, "completed" => 2 }
+  enum letting_in_sheltered_accomodation: { "Yes - sheltered housing" => 0, "Yes - extra care housing" => 1, "No" => 2, "Don't know" => 3 }
```

Comment:
Worth moving both of these to the CaseLog constants file so they're all in the same place?

Reply:
I've just updated this in efd1442406ba19dc6be8c35d6b905a1c4343a035, is that what you're expecting?

Reply:
Yep. Worth Paul reviewing but otherwise :ship: